nemo/collections/tts/parts/glow_tts.py

```diff
@@ -0,0 +1,606 @@
+import math
```

Comment:
I would consider splitting this file partly into helpers and partly into modules/submodules.py or another file inside of modules

Reply:
I thought that helpers was more for functions separate from the actual model, while "parts" in other collections seems to be used for submodules and functions for them? E.g. in asr parts folder contains both the parsers and the jasper submodules.

Reply:
I tried to have torch.nn.modules under modules inside TTS and all other "python" functions inside helpers

Reply:
I changed this to be a file in modules, but I don't think it makes sense to put functions that are directly relevant to the model in helpers

Reply:
Hmm, I have a bunch of model specific code inside of helpers. Should we rename helpers to parts?
The directory structure could definitely be improved if you have suggestions to make.